test/generators/UserAnswersEntryGenerators.scala

```diff
@@ -261,7 +261,9 @@ trait UserAnswersEntryGenerators {
       generateAuthorisationAnswers orElse
       generateLimitAnswers orElse
       generateCarrierDetailsAnswers orElse
-      generateEquipmentAnswers
+      generateEquipmentAnswers orElse
+      generateSealAnswers orElse
+      generateItemNumberAnswers
```

Comment:
Can we move these into `generateEquipmentAnswers`? Just keeps the section / sub-section structure consistent